build.json

```diff
@@ -405,6 +405,7 @@
         "src/core/httpcli/parser.h",
         "src/core/security/auth.h",
         "src/core/security/base64.h",
+        "src/core/security/context.h",
```

Comment:
bikeshed: can we choose a different name than context.{h,c} - visual studio only allows a unique filename to be used once per project, and I feel like context is one of those names that we'd want to be precious about